lib/generators/rodauth/views_generator.rb

```diff
@@ -96,11 +100,23 @@ def configuration_name
         # We need to use the *_tag helpers on versions lower than Rails 5.1.
         def form_helpers_compatibility(content)
           content
-            .gsub(/form_with url: (.+) do \|form\|/, 'form_tag \1 do')
-            .gsub(/form\.(label|submit)/, '\1_tag')
-            .gsub(/form\.(email|password|text|telephone|hidden)_field (\S+), value:/, '\1_field_tag \2,')
-            .gsub(/form\.radio_button (\S+), (\S+),/, 'radio_button_tag \1, \2, false,')
-            .gsub(/form\.check_box (\S+), (.+) /, 'check_box_tag \1, "t", false, \2 ')
+          .gsub(/form_with url: (.+) do \|form\|/, 'form_tag \1 do')
+          .gsub(/form\.(label|submit)/, '\1_tag')
+          .gsub(/form\.(email|password|text|telephone|hidden)_field (\S+), value:/, '\1_field_tag \2,')
+          .gsub(/form\.radio_button (\S+), (\S+),/, 'radio_button_tag \1, \2, false,')
+          .gsub(/form\.check_box (\S+), (.+) /, 'check_box_tag \1, "t", false, \2 ')
```

Comment:
Please undo the indentation change.

Reply:
[fixed](https://github.com/janko/rodauth-rails/pull/114/commits/ee23646bcdaa0641c17f0bf189001413b14c895b). 

The culprit was a 'beautify ruby' gem I was using that automatically formats on save. I'll watch for similar in case there is a reversion.